runtime/testy/studentCode.py

```diff
@@ -102,13 +113,11 @@ def createKey_main():
   restarts = Robot.getValue("Restarts")
   Robot.setValue(restarts+1, "Restarts")
 
-def hibikeSubscribeDevice_setup():
-  pass
+# def hibikeSubscribeDevice_setup():
```

Comment:
Another option is to define some mechanism (like a `@manual` decorator) to annotate tests such that they run with `-t <testname>` but not with plain `-t`.